README.md

```diff
@@ -157,7 +157,7 @@ $ cargo run --features json_example --example cli FILENAME.sql [--dialectname]
 
 This parser is currently being used by the [DataFusion] query engine, [LocustDB],
 [Ballista], [GlueSQL], [Opteryx], [Polars], [PRQL], [Qrlew], [JumpWire], [ParadeDB], [CipherStash Proxy],
-and [GreptimeDB].
+[Sqawk](https://github.com/jgarzik/sqawk) and [GreptimeDB].
```

Comment:
Hi @jgarzik! could you add this link at the bottom of the file, similar to the others?